Kinvey.Core/Core/NetworkFactory.cs

```diff
@@ -219,7 +219,26 @@ public NetworkRequest<T> BuildGetAggregateRequest<T>(string collectionName, Enum
 			return create;
 		}
 
-		public NetworkRequest<T> buildUpdateRequest <T> (string collectionName, T entity, string entityID)
+        /// <summary>
+        /// Creates a network request for multi insert operation.
+        /// </summary>
+        /// <returns>A created network request.</returns>
+        /// <param name="collectionName">A name of a collection.</param>
+        /// <param name="entities">Entities to insert.</param>
+        public NetworkRequest<U> buildMultiInsertRequest<T, U>(string collectionName, List<T> entities)
```

Comment:
Should this method name be capitalized to stay consistent with .NET coding conventions?

Reply:
@vinaygahlawat 
Yes, it should.
I came up with such name as the most of methods in the
https://github.com/Kinvey/dotnet-sdk/blob/feature/MLIBZ-3015_Implement_multi-record_insert_new/Kinvey.Core/Core/NetworkFactory.cs
have like naming.
I seemed to copy from some of them.
Anyway I agree with you. I am going to rename it now.
But in future, when we are going to do a task for improvement code legacy, we must rename other ones.